lib/mumukit/auth/permissions.rb

```diff
@@ -54,7 +54,13 @@ def granted_organizations_for(role)
   end
 
   def add_permission!(role, *grants)
-    scope_for(role).add_grant! *grants
+    role = role.to_mumukit_role
```

Comment:
No, it is intended, since `push_and_compact!` may be called from other places. On the other hand, calling this here avoids re-running the conversion within the `each` loop - since it will be a no-op.